customers.go

```diff
@@ -22,10 +25,30 @@ type Customer struct {
 	UpdatedAt time.Time
 }
 
+// Customers is a collection of customers
+type Customers []Customer
+
+// CustomerSearchQuery contains all the supported search queries
+type CustomerSearchQuery struct {
+	Tag string
```

Comment:
I think these should be like for like with the documentation? Also could take one for the team and quickly throw them all in here to save time in the future? Copy them in the same order as shown in the docs and then build them exactly like below in same order :)

Reason I say like for like is because I think usually everything in the package is just copied from the shopify documentation and its easier to track.

Reply:
Yeh, we thought this but when exactly would this be used again? The reason why we are using it is because we need to search by tag and you cannot do that via REST. 

Realistically we probably want to use GraphQL for instances where we want more than a single property. This search is not that useful. So although I agree it would be nice to add all the properties. I cannot see this method much use anyway in the future. It's just ideal for now.

Reply:
I think as it isn't much effort to do and would increase development if needed in the future it is worth just quickly doing. To save time I have just done this quickly :+1: